registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -62,6 +62,38 @@
           "name": "RMRK"
         }
       ]
+    },
+    "0x41fdc19610c6cd2c489e6cbb00881d871e14f6acfca798f2b4c99d6d7556fa95": {
+      "tokenId": "0x41fdc19610c6cd2c489e6cbb00881d871e14f6acfca798f2b4c99d6d7556fa95",
+      "deployer": "0x5E335399DEeaD358Fb893f6594F2A1eDC91C340f",
+      "originalMinter": null,
+      "prettySymbol": "CONK",
+      "decimals": 18,
+      "originAxelarChainId": "fantom",
+      "tokenType": "canonical",
+      "deploySalt": "0x",
+      "iconUrls": {
+        "svg": "https://raw.githubusercontent.com/axelarnetwork/axelar-configs/images/tokens/conk.svg"
```

Comment:
sorry another nit: can you change this to `https://raw.githubusercontent.com/axelarnetwork/axelar-configs/main/images/tokens/conk.svg`. 

we'll change the autogeneration of this upstream as well